internal/pkg/admission/policy-server_integration_test.go

```diff
@@ -1,95 +0,0 @@
-package admission
-
-import (
-	"context"
-	"crypto/tls"
-	"crypto/x509"
-	"net/http"
-	"sync"
-	"testing"
-	"time"
-
-	"github.com/kubewarden/kubewarden-controller/internal/pkg/admissionregistration"
-	"github.com/kubewarden/kubewarden-controller/internal/pkg/constants"
-)
-
-const port = "8181"
-
-func TestCAAndCertificateCreationInAHttpsServer(t *testing.T) {
```

Comment:
Should we drop this test?

Reply:
Fair enough, I've re added this test in the internal/pkg/admission/policy-server-ca-secret_test.go file. ;)